app/projects/template.hbs

```diff
@@ -0,0 +1,115 @@
+{{#if (not hasAccess)}}
+  <div class="text-center" data-test-access-forbidden>
+    <h1>Access forbidden</h1>
+    <h4>You do not have the permission to access this page</h4>
+  </div>
+{{else}}
+  {{#power-select
+    data-test-project-selection
+    tagName="div"
+    class="header"
+    options=projects
+    placeholder="Select project..."
+    searchField="name"
+    onchange=(action (queue (action (mut selectedProject)) (perform fetchTasksOfProject)))
+    selected=selectedProject
+    allowClear=true
+    as |project|
+  }}
+    {{project.name}}
+  {{/power-select}}
+  {{#if selectedProject}}
+    <h3>
+      <span class="header-left">
+        Tasks of {{selectedProject.name}}
+      </span>
+      <span class="header-right">
+        <button class="btn btn-primary" {{action (perform createTask)}} data-test-add-task>Add Task</button>
+      </span>
+    </h3>
+    <div style="height: 400px; margin-bottom: 20px">
```

Comment:
See if you can find/create a CSS class you can use for that.

Reply:
@fkm fixed